src/components/confirm-dialog/ConfirmDialog.vue

```diff
@@ -136,6 +136,20 @@ export default class ConfirmDialog extends Vue {
   }
 }
 </script>
+
 <style lang="scss" scoped>
-  @import '@/assets/styles/theme.scss';
+@import '@/assets/styles/theme.scss';
+
+.v-card__text {
+  color: $gray7 !important;
+}
+
+.v-btn {
+  text-transform: unset;
```

Comment:
I noticed that Storybook uses SASS and by default capitalizes all v-btn content. This allows us to see the buttons in their original case (ie, "Yes").